exercises/Perception-Driven_Manipulation/solution_ws/src/collision_avoidance_pick_and_place/src/services/target_recognition_service.cpp

```diff
@@ -13,7 +13,7 @@
 #include <sensor_msgs/PointCloud2.h>
 #include <sensor_msgs/point_cloud_conversion.h>
 #include <pcl_conversions/pcl_conversions.h>
-#include <pcl/ros/conversions.h>
+#include <pcl_conversions/pcl_conversions.h>
```

Comment:
double include?

Reply:
Good catch, I didn't port this file so not sure how I ended up making that change.  I could work on porting it but there won't be a way to test it unless we setup a sensor and the AR perception.  We can also delete it or just keep it in an "unported" state

Reply:
Didn't realize that it was already included